package.json

```diff
@@ -21,6 +23,27 @@
     "type": "git",
     "url": "git+https://github.com/rhysd/NyaoVim.git"
   },
+  "build": {
+    "asar": false,
+    "appId": "org.vim.NyaoVim",
```

Comment:
I think `neovim` is better than `vim` because Neovim is a different software from Vim.